p2p/ipld/read_test.go

```diff
@@ -8,172 +8,68 @@ import (
 	"math"
 	"math/rand"
 	"sort"
-	"strings"
 	"testing"
 	"time"
 
-	"github.com/ipfs/go-cid"
-	"github.com/ipfs/go-ipfs/core/coreapi"
-	coremock "github.com/ipfs/go-ipfs/core/mock"
 	format "github.com/ipfs/go-ipld-format"
-	iface "github.com/ipfs/interface-go-ipfs-core"
+	mdutils "github.com/ipfs/go-merkledag/test"
 	"github.com/lazyledger/nmt"
 	"github.com/lazyledger/nmt/namespace"
 	"github.com/lazyledger/rsmt2d"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/lazyledger/lazyledger-core/ipfs/plugin"
-	"github.com/lazyledger/lazyledger-core/p2p/ipld/racedetector"
 	"github.com/lazyledger/lazyledger-core/p2p/ipld/wrapper"
 	"github.com/lazyledger/lazyledger-core/types"
 	"github.com/lazyledger/lazyledger-core/types/consts"
 )
 
-func TestLeafPath(t *testing.T) {
-	type test struct {
-		name         string
-		index, total uint32
-		expected     []string
-	}
-
-	// test cases
-	tests := []test{
-		{"nil", 0, 0, []string(nil)},
-		{"0 index 16 total leaves", 0, 16, strings.Split("0/0/0/0", "/")},
-		{"1 index 16 total leaves", 1, 16, strings.Split("0/0/0/1", "/")},
-		{"9 index 16 total leaves", 9, 16, strings.Split("1/0/0/1", "/")},
-		{"15 index 16 total leaves", 15, 16, strings.Split("1/1/1/1", "/")},
-	}
-
-	for _, tt := range tests {
-		tt := tt
-		t.Run(tt.name, func(t *testing.T) {
-			result, err := leafPath(tt.index, tt.total)
-			if err != nil {
-				t.Error(err)
-			}
-			assert.Equal(t, tt.expected, result)
-		},
-		)
-	}
-}
-
-func Test_isPowerOf2(t *testing.T) {
-	type test struct {
-		input    uint32
-		expected bool
-	}
-	tests := []test{
-		{
-			input:    2,
-			expected: true,
-		},
-		{
-			input:    11,
-			expected: false,
-		},
-		{
-			input:    511,
-			expected: false,
-		},
-
-		{
-			input:    0,
-			expected: true,
-		},
-		{
-			input:    1,
-			expected: true,
-		},
-		{
-			input:    16,
-			expected: true,
-		},
-	}
-	for _, tt := range tests {
-		res := isPowerOf2(tt.input)
-		assert.Equal(t, tt.expected, res, fmt.Sprintf("input was %d", tt.input))
-	}
-}
```

Comment:
Did you delete these two tests because they test private helpers?

Reply:
I delete tests as I deleted those helpers - there are no more needed.

Reply:
Oh, I see. NVM.